pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -387,6 +387,10 @@ type CertificateCondition struct {
 	// transition, complementing reason.
 	// +optional
 	Message string `json:"message,omitempty"`
+
+	// ObservedGeneration is the generation of this Certificate when this field
+	// is set.
```

Comment:
Can we utilise the comments from the upstream conditions KEP? 😄 (and ensure tags/field types align)

Reply:
Would it be worth adding a link to the upstream KEP too?

i.e.
https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1623-standardize-conditions

It's always great to be able to click on a link to "see more": why observedGeneration exists, how it can be used...